### DISCUSSION TITLE
complete NOOB

### DISCUSSION THREAD

**Discussion by skyebits** [community/community#72876] [Programming Help]
I'm learning R(first programming language for now), and I'm trying to link the rstudio to an Existing project.. but I actually can't find a guide, especially the BASH portion which has quickly become overwhelming because I dunno WTF I'm looking at. 

I picked a directory path in a basic place(made the repository) but just got lost after that. So I got a bunch of useless repositories (BTW, how to delete it?)  The BASH tutorial would be nice, as I have no clue at all. coding suddenly seems easier to do then using this platform.

Thank you for even taking the time to even read this post, any advice you can offer is greatly appreciated.

**Comment by deshaXii** [comment]
lol

**Comment by 4lexxe** [comment]
The question is, why are you studying a programming language like R as your first programming language? XD

There's nothing wrong with it, but if you're already finding it challenging to locate a guide, I suggest going for a programming language that's similar.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @skyebits ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
